lib/events/api.go

```diff
@@ -841,6 +841,44 @@ type SessionStreamer interface {
 	StreamSessionEvents(ctx context.Context, sessionID session.ID, startIndex int64) (chan apievents.AuditEvent, chan error)
 }
 
+type SearchEventsRequest struct {
+	// FromUTC is oldest date of returned events, can be zero.
+	FromUTC time.Time
+	// ToUTC is the newest date of returned events.
+	ToUTC time.Time
```

Comment:
I realize that this is just a structification of the arguments to `SearchEvents`, but could we also rename these to `From` and `To` or some other names? `time.Time` carries timezone info (and in a lot of places in this PR is set to `time.Now().Add(something)` which is not generally UTC).

Reply:
reworked in https://github.com/gravitational/teleport/pull/26433/commits/0a9bf4ccb890ec1c0a638f716bb30a53f8769ef8